docs/2-the-zarf-cli/100-cli-commands/zarf_prepare.md

```diff
@@ -27,6 +27,7 @@ Tools to help prepare assets for packaging
 * [zarf](zarf.md)	 - DevSecOps for Airgap
 * [zarf prepare find-images](zarf_prepare_find-images.md)	 - Evaluates components in a zarf file to identify images specified in their helm charts and manifests
 * [zarf prepare generate-config](zarf_prepare_generate-config.md)	 - Generates a config file for Zarf
+* [zarf prepare lint](zarf_prepare_lint.md)	 - Tools for validating schema / ensuring best practices
```

Comment:
verifies the package schema, makes sure there isn't stuff in the yaml zarf will ignore.